src/rest-server/src/routes/v2/user.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// module dependencies
+const express = require('express');
+const token = require('../../middlewares/token');
+const userController = require('../../controllers/v2/user');
+const userInputSchema = require('../../config/v2/user');
+const param = require('../../middlewares/parameter');
+const jobRouter = require('../job');
+const authnConfig = require('../../config/authn');
+
+const router = new express.Router();
+
+router.route('/get/:username/')
+  /** Get /api/v2/user/get/:username */
+  .get(token.check, userController.getUser);
+
+router.route('/get/')
```

Comment:
The `/get/` verb should not be included in rest api's url

Reply:
Done